Dockerfile.envoy

```diff
@@ -1,6 +1,6 @@
 FROM debian:sid as builder
 
-ADD qat1.7.l.4.3.0-00033.tar.gz .
+ADD qat1.7.l.4.4.0-00023.tar.gz .
```

Comment:
seems that version is not available to download: 

https://01.org/sites/default/files/downloads/intelr-quickassist-technology/qat1.7.l.4.4.0-00023.tar.gz

Reply:
@juliocri the url is different for that version https://01.org/sites/default/files/downloads/qat1.7.l.4.4.0-00023.tar.gz that's why I have commented [these lines ](https://github.com/electrocucaracha/kubernetes-qat-envoy/blob/clearlinux/vagrant/krd-vars.yml#L19-L20)

Reply:
yes :/ sorry about it, my bad.